code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -94,6 +99,19 @@
 		card_slot.try_eject(, usr)
 
 // Eject ID card from computer, if it has ID slot with card inside.
+/obj/item/device/modular_computer/proc/eject_card()
+	set name = "Eject Intellicard"
+	set category = "Object"
+	set src in view(1)
+
+	if(issilicon(usr))
+		return
+	var/obj/item/weapon/computer_hardware/ai_slot/ai_slot = all_components[MC_AI]
+	if(usr.canUseTopic(src))
+		ai_slot.try_eject(, usr,1)
```

Comment:
I'm calling the try eject proc?
When whoever it was rewrote hardware, they had the id slot work this way.